bytestring_splitter/__init__.py

```diff
@@ -47,7 +47,15 @@ def __init__(self, *message_parameters):
                     """You can't specify the length of the message as a direct argument to the constructor.
                     Instead, pass it as the second argument in a tuple (with the class as the first argument)""")
 
-    def __call__(self, splittable, return_remainder=False, msgpack_remainder=False):
+    def __call__(self, splittable: bytes, return_remainder=False, msgpack_remainder=False, single=False):
```

Comment:
Why not `multiple`?

Reply:
Well, which is the more sane default?

The machinery here is a *splitter*, meaning that the bytestring contains, as a high-level gesture, multiple pieces of knowledge.  Now, there's a strange case where you know that you have exactly one piece of knowledge, but you want to use the splitter anyway because it's part of your modeling (as with the kwargifier).

So I think that having to specify `single` but not `multiple` makes sense.

Reply:
:+1: